Source/ChocolateyGui/ViewModels/SourcesViewModel.cs

```diff
@@ -90,7 +90,7 @@ protected override void OnViewReady(object view)
         {
             Observable.FromEventPattern<PropertyChangedEventArgs>(this, nameof(PropertyChanged))
                 .Where(p => p.EventArgs.PropertyName == nameof(ActiveItem))
-                .Subscribe(p => DisplayName = $"Source - {ActiveItem.DisplayName}");
+                .Subscribe(p => DisplayName = $"Source - {ActiveItem?.DisplayName}");
```

Comment:
I got so annoyed by getting a null exception every time I closed the program, so just had to add it.